src/components/patient/AddressPanel.tsx

```diff
@@ -28,6 +54,9 @@ const AddressPanel: FC<{ data: PatientRecord }> = ({ data }) => {
         rows={[data]}
         autoHeight={true}
         hideFooter={true}
+        processRowUpdate={(newRow, oldRow) => onRowUpdate(newRow, oldRow)}
+        // TO-DO: handle errors
```

Comment:
```suggestion
        // TO-DO: handle errors
```
@IlyesBenAmara I wonder what this todo is about ?